gui/src/Console/State.purs

```diff
@@ -12,8 +12,8 @@ import Global.Effects
 
 initialState :: Channel Action -> State
 initialState chan =
-    { buffer : ">"
-    , display : ">"
+    { buffer : ""
+    , display : ""
```

Comment:
Why did we change this? Now when one opens the console they don't see anything. I think it's kinda misleading for a user to do so.